src/internal/requestAdapter.ts

```diff
@@ -50,6 +50,27 @@ export class RequestAdapter implements IRequestAdapter {
   constructor(private fetch: Fetch) {}
 
   public execute(uri: string, opt: IRequestOptions): Promise<any> {
+
+    /* TODO mock response, remove after having BE ready */
+    if (opt.method === Methods.POST && uri.includes("/ds/r/")) {
+      return new Promise((resolve, reject) => {
+        let records = opt.body;
+
+        if (!Array.isArray(records)) {
+          reject({
+            errors: ["Incorrect records array"]
+          });
+        } else {
+          resolve(records.map((record: any) => ({
+            id: "some random mocked id",
+            created_at: new Date().toJSON(),
+            updated_at: new Date().toJSON(),
+            ...record
```

Comment:
Isn't it better to have this at the top? If record ever has `id`, `created_at` or `updated_at` will replace the other values.